readme.md

```diff
@@ -119,15 +122,16 @@ What is the difference between the two following lines of code?
 @artist.save!
 ```
 
-> Your answer...
+> The second command with the ! bang will commit the save. Similar to a -f force command.
 
 ## Question 10
 
 Using jQuery, write an AJAX request to `http://tunr.com/artists` that would create a new artist with the name of 'Resin Laying Deer Figurine, Gold', and pop up a box saying "All done!" when complete.
 
 ```js
-// Your code starts here...
-
+$(.create).artist(name:"Resin Laying Deer Figurine, Gold");
+  prompt("All done!");
+forgot the AJAX link syntax.
 ```
 
```

Comment:
You should definitely review the basic structure of an AJAX call. Included a possible answer below. Let me know if there's anything I can do to help out with us - will check in about it when we're back from break.

```js
$.ajax({
  method: "POST",
  url: "http://tunr.com/artists",
  dataType: "json",
  data: {
    name: "Resin Laying Deer Figurine, Gold"
  }
}).done(function(){
  alert("All done!");
});
```